composer.json

```diff
@@ -14,7 +14,7 @@
     },
     "require": {
         "php": ">= 8.0",
-        "silverstripe/cms": "^4"
+        "silverstripe/cms": "^4 || ^5"
```

Comment:
Can we have just version 5 :) if you need both update Actions to test on both versions

Reply:
Cool, thanks for the feedback! I've updated to only have version 5